src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx

```diff
@@ -25,7 +25,7 @@ export function TestRunSpecDetails() {
   if (!spec) {
     return (
       <div className="flex h-full w-full items-center justify-center p-2">
-        <div className={styles.standardMessaging}>Select a test to see its details here</div>
+<div className={styles.standardMessaging}>Select a test to see its details here</div>
```

Comment:
The indent seems not right. Can you run prettier on it?

Reply:
Thanks! Looks like it was a GitHub merge thing, should be good now.